react/components/Totalizer/index.js

```diff
@@ -68,11 +70,26 @@ class Totalizer extends PureComponent {
                   </div>
                 ) : (
                   <>
+                    {item.inverted && (
+                      <div className={`mb${invertedMargin}`}>
```

Comment:
Why not just 
```suggestion
                      <div className={`${item.inverted ? 'mb' : 'mt'}${invertedMargin}`}>
``` ?

Reply:
This way you can avoid repeating the code down below:

Reply:
Harder to read in my opinion and "you're already within" the `{item.inverted && ` condition.

Reply:
Yeah, it makes sense.